spec/helpers/ajax-helper.ts

```diff
@@ -218,4 +226,33 @@ export class MockXMLHttpRequest {
       }
     });
   }
-}
\ No newline at end of file
+}
+
+export class MockXMLHttpRequestInternetExplorer extends MockXMLHttpRequest {
+  constructor() {
+    super();
+  }
+
+  private mockHttp204() {
+    this.responseType = '';
+    this.responseText = '';
+    this.response = '';
+  }
+
+  protected jsonResponseValue(response: any) {
+    if (this.status == 204) {
+      this.mockHttp204();
+      return;
+    }
+    return super.jsonResponseValue(response);
+  }
+
+  protected defaultResponseValue() {
```

Comment:
I just pulled down and while trying it, seems this specific (`MockXMLHttpRequestInternetExplorer::defaultResponseValue()`) hasn't executed. Maybe I'm missing something?